data/sql_updates/totals_pac.sql

```diff
@@ -4,84 +4,106 @@ create materialized view ofec_totals_pacs_mv_tmp as
+        sum(pnp.ttl_fed_elect_actvy_per) as fed_election_activity,
+        sum(pnp.ttl_fed_op_exp_per) as fed_operating_expenditures,
+        sum(pnp.ttl_fed_receipts_per) as fed_receipts,
+        sum(pnp.indt_exp_per) as independent_expenditures,
+        sum(pnp.indv_contb_ref_per) as refunded_individual_contributions,
+        sum(pnp.indv_item_contb_per) as individual_itemized_contributions,
+        sum(pnp.indv_unitem_contb_per) as individual_unitemized_contributions,
+        sum(pnp.ttl_indv_contb) as individual_contributions,
+        sum(pnp.loan_repymts_made_per) as loan_repayments_made,
+        sum(pnp.loan_repymts_received_per) as loan_repayments_received,
+        sum(pnp.loans_made_per) as loans_made,
+        sum(pnp.net_contb_per) as net_contributions,
+        sum(pnp.net_op_exp_per) as net_operating_expenditures,
+        sum(pnp.non_alloc_fed_elect_actvy_per) as non_allocated_fed_election_activity,
+        sum(pnp.ttl_nonfed_tranf_per) as total_transfers,
+        sum(pnp.ttl_op_exp_per) as offsets_to_operating_expenditures,
+        sum(pnp.ttl_op_exp_per) as operating_expenditures,
+        sum(pnp.other_disb_per) as other_disbursements,
+        sum(pnp.other_fed_op_exp_per) as other_fed_operating_expenditures,
+        sum(pnp.other_fed_receipts_per) as other_fed_receipts,
+        sum(pnp.other_pol_cmte_contb_per_i) as other_political_committee_contributions,
+        sum(pnp.other_pol_cmte_refund) as refunded_other_political_committee_contributions,
+        sum(pnp.pol_pty_cmte_contb_per_i) as political_party_committee_contributions,
+        sum(pnp.pol_pty_cmte_refund) as refunded_political_party_committee_contributions,
+        sum(pnp.ttl_receipts) as receipts,
+        sum(pnp.shared_fed_actvy_fed_shr_per) as shared_fed_activity,
+        sum(pnp.shared_fed_actvy_nonfed_per) as shared_fed_activity_nonfed,
+        sum(pnp.shared_fed_op_exp_per) as shared_fed_operating_expenditures,
+        sum(pnp.shared_nonfed_op_exp_per) as shared_nonfed_operating_expenditures,
+        sum(pnp.tranf_from_affiliated_pty_per) as transfers_from_affiliated_party,
+        sum(pnp.tranf_from_nonfed_acct_per) as transfers_from_nonfed_account,
+        sum(pnp.tranf_from_nonfed_levin_per) as transfers_from_nonfed_levin,
+        sum(pnp.tranf_to_affliliated_cmte_per) as transfers_to_affiliated_committee,
+        max(last.rpt_tp_desc) as last_report_type_full,
+        max(last.begin_image_num) as last_beginning_image_number,
+        max(last.coh_cop) as last_cash_on_hand_end_period,
+        max(last.debts_owed_by_cmte) as last_debts_owed_by_committee,
+        max(last.debts_owed_to_cmte) as last_debts_owed_to_committee,
+        max(last.rpt_yr) as last_report_year
+    from
+        fec_vsum_f3x_vw pnp
+        inner join ofec_committee_detail_mv_tmp comm_dets on cmte_id = comm_dets.committee_id
+        left join last using (cmte_id, election_cycle)
+    where
+        pnp.most_recent_filing_flag like 'Y'
```

Comment:
This was already in the code- and I was probably the one to put it there- but I we can use `=` instead of `like` because it is faster. (We could also address that detail later)